src/GoodsList.tsx

```diff
@@ -5,12 +5,20 @@ type Props = {
   goods: Good[]
 };
 
-export const GoodsList: React.FC<Props> = ({ goods }) => (
-  <ul>
-    {goods.map(good => (
-      <li key={good.id} data-cy="good">
-        {good.name}
-      </li>
-    ))}
-  </ul>
-);
+export const GoodsList: React.FC<Props> = React.memo(({ goods }) => (
+  <div className="content">
+    <ul>
+      {goods.map(good => (
+        <div className="">
```

Comment:
Avoid using an empty className attribute. Either remove it or add a class name 🧹